app/src/main/java/com/akoscz/youtube/YouTubeRecyclerViewFragment.java

```diff
@@ -218,6 +230,14 @@ private void handleGetPlaylistResult(PlaylistVideos playlistVideos, Pair<String,
         playlistVideos.addAll(result.second);
         mPlaylistCardAdapter.notifyItemRangeInserted(positionStart, result.second.size());
     }
+    
+    private void showProgressDialog(Boolean shouldShould){
```

Comment:
There is really no need for this helper method.  The code would be more readable by simply calling the show() and dismiss() methods on the progress bar inline rather than abstracting it into this helper method.